### PR TITLE
Log in page

### PR DESCRIPTION
**Pull request by vikawal**
This is pull-request mainly about log-in page, but also NAME field from register page was deleted. Please, check it. Especially JS part =-))

### CODE REVIEW

File: client/components/pageLogIn/logIn.mjs
Comment (TatianaBortnik) [inline client/components/pageLogIn/logIn.mjs:23]: All commented code should be deleted

**Review by TatianaBortnik — CHANGES_REQUESTED** [review]

File: client/components/pageLogIn/logIn.mjs
Comment (vikawal) [reply client/components/pageLogIn/logIn.mjs]: Ok.Done

**Review by tbazko — APPROVED** [review]

**Comment by vikawal** [comment]
should I merge it or wait for second approval?

**Review by anynikulina — APPROVED** [review]

**Pull request merged**
